cmd/xl-v1-object_test.go

```diff
@@ -189,7 +195,7 @@ func TestGetObjectNoQuorum(t *testing.T) {
 	// Make 9 disks offline, which leaves less than quorum number of disks
 	// in a 16 disk XL setup. The original disks are 'replaced' with
 	// naughtyDisks that fail after 'f' successful StorageAPI method
-	// invocations, where f - [0,2)
+	// invocations, where f - [0,2]
```

Comment:
By `[0,2)` they meant half open interval, including 0, but not 2 - meaning just 0 or 1; see https://en.wikipedia.org/wiki/Interval_(mathematics)#Terminology - so if we don't want to use such deep-math notation, it should be written `[0,1]` here as we mean `f` can be 0 or 1.